backend/manager/modules/dal/src/main/resources/bundles/AuditLogMessages.properties

```diff
@@ -482,6 +482,12 @@ USER_REMOVE_ROLE=Role ${RoleName} removed from ${UserName}
 USER_REMOVE_ROLE_FAILED=Failed to remove role ${RoleName} (User ${UserName})
 USER_UPDATE_PROFILE=Updated profile for ${UserName}
 USER_UPDATE_PROFILE_FAILED=Failed to update profile for ${UserName}
+USER_PROFILE_CREATE_PROPERTY=Account Settings: ${UserName} created property '${ProfileProperty}' for user ${ProfileUser}.
+USER_PROFILE_CREATE_PROPERTY_FAILED=Account Settings: ${UserName} failed to create property '${ProfileProperty}' for user ${ProfileUser}.
+USER_PROFILE_REMOVE_PROPERTY=Account Settings: ${UserName} removed property '${ProfileProperty}' for user ${ProfileUser}.
+USER_PROFILE_REMOVE_PROPERTY_FAILED=Account Settings: ${UserName} failed to remove property '${ProfileProperty}' for user ${ProfileUser}.
+USER_PROFILE_REPLACE_PROPERTY=Account Settings: ${UserName} replaced property '${ProfileProperty}' for user ${ProfileUser}.
+USER_PROFILE_REPLACE_PROPERTY_FAILED=Account Settings: ${UserName} failed to replace property '${ProfileProperty}' for user ${ProfileUser}.
```

Comment:
> When you write replace, it's not clear if replaced by what.
>

Replaced by another property with the same name. 

> Maybe using the term 'update' is better here since what's really interesting is that the value was updated.
>

We cannot use `update` as this suggest mutating the state of the same entity - in this case a new immutable entity is created.

Reply:
I know, but users won't surely understand WDYM by replace if they won't dig into DB queries, so maybe rephrasing to something  like:
 "userxx regenerated (for updating) property xxx for user...."

Reply:
`regenerated (for updating)` suggest mutation which is incorrect since a new (immutable) entity is created.